README.md

```diff
@@ -1,66 +1,44 @@
-# landr
+<h1 align="center">
+    Landr
+</h1>
 
```

Comment:
A short explanation fo what landr does would be ideal here

Reply:
The Readme needs some love for sure. For now, I added the previous subtitle "Build a website for your software projects with one command.".